README.md

```diff
@@ -1,259 +1,16 @@
 # ForgAE
 
+[![npm version](https://badge.fury.io/js/forgae.svg)](https://badge.fury.io/js/forgae)
+
 **ForgAE** is an aeternity framework which helps with setting up a project.
 The framework makes the development of smart contracts in the aeternity network pretty easy. It provides commands for compilation, deployment of smart contracts, running a local node, local compiler and unit testing the contracts.
 
-The framework can be installed via npm:
-```
-npm i -g forgae
-```
-
-## Initialize Forgae
-
-```
-
-forgae init
-
-```
-
-The **init** command creates aeternity project structure with a few folders
-in which the developer can create
-
-the contracts, tests and deployment files and scripts. Docker configuration
-files are also created, for easy use of the aeternity blockchain network.
-
-The **init --update** command updates projects files. Important all files in docker folder and docker-compose will be replaced.
-
-## Start your local development node
+### Installing
 
+```text
+npm i -g forgae-cli
```

Comment:
This is not correct, `npm i -g forgae` as we are not changing the name of the CLI project